third_party/intel/lib/TritonIntelGPUTransforms/AccelerateMatmul.cpp

```diff
@@ -10,6 +10,9 @@
 #include "triton/Dialect/TritonGPU/IR/Dialect.h"
 #include "llvm/ADT/TypeSwitch.h"
 
+#define PVC_2D_LOAD_MAXIMUM_NUMBER_OF_ROWS 32
+#define PVC_2D_LOAD_MAXIMUM_BYTES_OF_COLS 64
```

Comment:
```suggestion
static constexpr unsigned pvc2DLoadMaximumNumberOfRows = 32;
static constexpr unsigned pvc2DLoadMaximumBytesOfCols = 64;
```

What does "bytes of cols" mean?

Reply:
Total number of bytes per row. Does `TotalNumberBytesPerRow` make more sense?

Reply:
Yeah, I think so!